lib/main.dart

```diff
@@ -148,6 +180,18 @@ class MyAppState extends State<MyApp> {
             routerConfig: router,
             theme: theme,
             title: '$titlePrefix Wallet',
+            locale: _locale,
+            localizationsDelegates: [
+              AppLocalizations.delegate, // Add this line
+              GlobalMaterialLocalizations.delegate,
```

Comment:
similar comment with regards to material. We don't use or need material

Reply:
removed